utils/parts.stanza

```diff
@@ -1003,7 +1010,7 @@ defn to-jitx (lp: LandPatternCode, jitx-pads-by-pcb-pad-name: HashTable<String,
 
     do(to-jitx, layers(lp))
     do(to-jitx, geometries(lp))
-    do(to-jitx, model3ds(lp))
+    do(to-jitx{model3d(_)}, model3ds(lp))
```

Comment:
I actually added function composition to `lang-utils` 4 months ago to allow for `do(to-jitx * model3d, model3ds(lp))`

Reply:
This?
https://github.com/JITx-Inc/jitx-client/blob/3c5edc95098a6b65aa20b24e79ac635685be82c0/jitpcb/src/utils/lang-utils.stanza#L718-L719
Sweet!

Reply:
You got me